src/blocks/SliderNew/Slider.scss

```diff
@@ -0,0 +1,278 @@
+@import '../../../styles/mixins';
+@import '../../../styles/variables';
+
+$slideOffset: 8px;
```

Comment:
we are use this variable in one place, I think we can use in place and use `$indent[...]` variable